.github/workflows/CI-distro.yml

```
@@ -0,0 +1,110 @@
+          - gap-package: 'guarana'                    # `AL_EXECUTABLE, the executable for PARI/GP, has to be set`
+          - gap-package: 'hap'                        # `polymake command not found. Please set POLYMAKE_COMMAND by hand`
+          - gap-package: 'hapcryst'                   # `polymake command not found. Please set POLYMAKE_COMMAND by hand`
+          - gap-package: 'io'                         # segfaults, most likely due to child process handling
+          - gap-package: 'itc'                        # dependency `xgap` has no jll
+          - gap-package: 'localizeringforhomalg'      # `Error, found no GAP executable in PATH`
+          - gap-package: 'normalizinterface'          # Tests fail: NormalizInterface currently bundles Normaliz 3.9.3 and its test suite is tuned to that, but our JLL builds it against normaliz_jll which has 3.10.2
+          #- gap-package: 'packagemanager'             # test failure
+          - gap-package: 'polenta'                    # `AL_EXECUTABLE, the executable for PARI/GP, has to be set`
+          - gap-package: 'polycyclic'                 # test suite currently broken and also disabled by GAP package distribution
+          - gap-package: 'polymaking'                 # `polymake command not found. Please set POLYMAKE_COMMAND by hand`
+          - gap-package: 'profiling'                  # segfaults during testing
+          - gap-package: 'radiroot'                   # `AL_EXECUTABLE, the executable for PARI/GP, has to be set`
+          - gap-package: 'ringsforhomalg'             # `Error, found no GAP executable in PATH`
+          - gap-package: 'semigroups'                 # no jll
+          #- gap-package: 'typeset'                    # needs `dot2tex` executable
+          #- gap-package: 'wpe'                        # ???
+          - gap-package: 'xgap'                       # no jll
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: "Install extra dependencies"
+        run: sudo apt-get install --no-install-recommends texlive-latex-base texlive-latex-recommended texlive-latex-extra texlive-fonts-recommended
+      - name: "Set up Julia"
+        uses: julia-actions/setup-julia@v2
+        with:
+          version: ${{ matrix.julia-version }}
+          arch: ${{ matrix.julia-arch }}
+      - name: "Cache artifacts"
+        uses: julia-actions/cache@v2
+        with:
+          include-matrix: false # don't create a cache per GAP package
+          cache-scratchspaces: false
+      - name: "Build package"
+        uses: julia-actions/julia-buildpkg@v1
+      - name: "Run GAP package tests"
+        run: |
+          JULIA_DEBUG=GAP julia --color=yes --project=. -e 'using GAP'  # ensure initialization
+          ls -l /home/runner/.julia/scratchspaces/*/binarywrappers_v1.11/bin
+          JULIA_DEBUG=GAP julia --color=yes --project=. -e 'using GAP, Test; @test GAP.Packages.test("${{ matrix.gap-package }}")'
```

We could drop `JULIA_DEBUG=GAP` here. Then again it might be useful for debugging CI failures in the future? Of course that could also be set about other CI tests, so perhaps not...
```suggestion
          julia --color=yes --project=. -e 'using GAP, Test; @test GAP.Packages.test("${{ matrix.gap-package }}")'
```